src/systemcmds/topic_listener/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ px4_add_module(
 	SRCS
 		topic_listener.cpp
 	DEPENDS
+		platforms__common
 		generate_topic_listener
 	)
-
+# vim: set noet ft=cmake fenc=utf-8 ff=unix :
```

Comment:
Not needed. This are used inconsistently, so unless someone wants to fix it entirely we're better off without.

Reply:
this can be removed, inherited from previous version.